daemon/src/model/cfd.rs

```diff
@@ -463,14 +464,14 @@ impl Cfd {
         initial_tx_fee_rate: TxFeeRate,
     ) -> Self {
         let long_initial_margin = calculate_long_margin(initial_price, quantity, leverage);
-        // TODO: Use FundingFee::default() if we don't want to charge funding fees
-        // based on quantity for the first settlement interval.
-        let funding_fees = FundingFee::new(
+
+        // TODO: Evaluate if we want to do that
+        let initial_funding_fee = calculate_funding_fee(
             long_initial_margin,
             initial_funding_rate,
             SETTLEMENT_INTERVAL.whole_hours(),
         )
-        .expect("values stored in db to be sane");
+        .expect("values stored in db to be sane"); // TODO: This expect is a bit weird
```

Comment:
I've looked at `calculate_funding_fee` and I think the expect makes sense:

- Division by zero can't happen given that `SETTLEMENT_INTERVAL` is the dividend.
- I think overflow can't happen on the first division since we're dividing by a number >= 1.0.
- We should be able to represent the funding fee with a `u64`. Otherwise our `margin * funding_fee * proportion > 18446744073709551615`, which is a lot of bitcoin.

Reply:
yes, that's exactly why I added an `expect()` in the first place.

Reply:
I removed the todo and the logic into a separate function that is used in the projection and the model to make it more clear that this is duplicated code.